src/Models/Rate.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Bookings\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Rate extends Model
+{
+    use HasFactory;
+
+    protected $guarded = ['id'];
+    protected $casts = [
+    ];
+
+    protected static function boot()
+    {
+        parent::boot();
+
+        static::creating(function ($rate) {
+            if (auth()->check()) {
+                $rate->creator_id = auth()->id();
+            }
+        });
+
+        static::saving(function ($rate) {
+            if (auth()->check()) {
+                $rate->updater_id = auth()->id();
+            }
+            //Todo: Implement BookingRateInterface
+//            if (empty($rate->getSlug())) {
```

Comment:
Please add them as methods and comment inside method:
```
function getSlug() {
    // @todo ...
}
```

Reply:
https://github.com/tipoff/bookings/pull/47/commits/72c7d932e5cbc54b28adf4217856e6da7357281f